lib/defopt.py

```diff
@@ -196,14 +202,21 @@ def run(funcs: Union[Callable, List[Callable], Dict[str, Callable]], *,
         dashes) to letters, to use as alternative short flags.  Defaults to
         `None`, which means to generate short flags for any non-ambiguous
         option.  Set to ``{}`` to completely disable short flags.
+    :param options:
```

Comment:
Upong further consideration, "options" is too vague a name, let's go for "cli_options" (unless you have a better proposal)